gef-ui/server/middleware/validateMongoId/index.js

```diff
@@ -0,0 +1,21 @@
+const validator = require('validator');
+const { HttpStatusCode } = require('axios');
+const { API_ERROR_CODE } = require('@ukef/dtfs2-common');
+
+/**
+ * Validator for a path parameter which is a mongo id
+ * @param {string} paramName
+ * @returns {import('express').RequestHandler}
+ */
+const validateMongoId = (paramName) => (req, res, next) => {
```

Comment:
Can this please be moved to `libs/common` as this function is highly re-usable. Thanks

Reply:
Thank you.